.tool-versions

```diff
@@ -1,2 +1,2 @@
-elixir 1.7.4
+elixir 1.7.4-otp-20
```

Comment:
This is to start using the version used on Travis, I presume? Looks like we're using elixir precompiled with otp 20 there indeed:
```
$ elixir --version
Erlang/OTP 20 [erts-9.3] [source] [64-bit] [smp:2:2] [ds:2:2:10] [async-threads:10] [hipe] [kernel-poll:false]
Elixir 1.7.4 (compiled with Erlang/OTP 20)
```
If that's the goal we can also set erlang to 20.3 in the following line

Reply:
It's not for Travis, it's for asdf. I think elixir 1.7.4 comes compiled for OTP 21 by default, which causes it to crash immediately.

So you have to explicitly use as compiled with otp-20 for it to work at all.

Reply:
I just checked, asdf elixir 1.7.4 comes precompiled using OTP 19:
```
elixir --version
Erlang/OTP 21 [erts-10.3.1] [source] [64-bit] [smp:6:6] [ds:6:6:10] [async-threads:1] [hipe]

Elixir 1.7.4 (compiled with Erlang/OTP 19)
```
This shouldn't be a problem, I didn't see any crashes myself? But we could make this change I think, it will make the version the same as on travis